Mage.Client/release/startClient.sh

```diff
@@ -1,3 +1,8 @@
 #!/bin/sh
 
-java -Xms256m -Xmx512m -XX:MaxPermSize=256m -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled -jar ./lib/mage-client-${project.version}.jar &
\ No newline at end of file
+script_dir="$(readlink -f -- "$BASH_SOURCE")"
+script_dir="$(dirname -- $script_dir)"
+cd $script_dir
+
+java -Xms256m -Xmx2g -XX:MaxPermSize=256m -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled -jar "${script_dir}/lib/mage-client-${project.version}.jar"
```

Comment:
Would make this a bit more readable and make future diffs smaller if you broke these out into lines.

```suggestion
java \
  -Xms256m \
  -Xmx2g \
  -XX:MaxPermSize=256m \
  -XX:+UseConcMarkSweepGC \
  -XX:+CMSClassUnloadingEnabled \
  -jar "${script_dir}/lib/mage-client-${project.version}.jar"
```